docs/community_resources/phpstorm_plugin.md

```diff
@@ -0,0 +1,171 @@
+- file name
+- directory
+
+![File template](img/phpstorm_plugin_file_template.png)
+
+You can customize file templates in **File** > **Settings** > **Editor** > **File and Code templates**.
+
+!!! tip
+
+    For more information about file templates, see [JetBrains documentation](https://www.jetbrains.com/help/phpstorm/settings-file-and-code-templates.html).
+
+### Live templates
+
+The plugin provides the following built-in live templates in Twig files:
+
+| Abbreviation | Comment |
+|---|---|
+| `ezcn` | `ez_content_name` |
+| `ezfd` | `ez_field_description` |
+| `ezfd?` | `ez_field_description` wrapped in an `ez_field_is_empty` check |
+| `ezfn` | `ez_field_name` |
+| `ezfn?` | `ez_field_name` wrapped in an `ez_field_is_empty` check |
+| `ezrc` | `ez_render_content` |
+| `ezrcq` | `ez_render_content_query` |
+| `ezrf` | `ez_render_field` |
+| `ezrf?` | `ez_render_field` wrapped in an `ez_field_is_empty` check |
+| `ezrl` | `ez_render_location` |
+| `ezrlq` | `ez_render_location_query` |
+
+You can customize live templates in **File** > **Settings** > **Editor** > **Live Templates**.
+
+!!! tip
+
+    For more information about live templates, see [JetBrains documentation](https://www.jetbrains.com/help/idea/using-live-templates.html).
+
+### Autocompletion in configuration files
+
+Plugin provides autocompletion for Ibexa DXP configuration structure in YAML files placed in `config/packages/`.
+
+Besides configuration structure, for the following YAML keys addition suggestions are available:
+
+- List of available view matchers, for:
+    - `ezplatform.<scope>.content_view.<view_type>.<view_name>.match`
+    - `ezplatform.<scope>.content_create_view.<view_type>.<view_name>.match`
+    - `ezplatform.<scope>.content_edit_view.<view_type>.<view_name>.match`
+    - `ezplatform.<scope>.content_translate_view.<view_type>.<view_name>.match`
+-  List of available SiteAccess matchers, for:
+    - `ezplatform.siteaccess.match`
+- List of available block attribute types, for:
+    - `ezplatform_page_fieldtype.blocks.<block_name>.attributes.<attribute_name>.type`
+
+### Structure autocompletion in DBAL schema file
+
+Autocompletion is also available for DBAL schema file structure.
+
+To enable autocompletion, you must place the file in the `config` directory and name it `schema.yaml`.
+
+### Dynamic settings autocompletion
+
+Parameter names suggestions are available in `\eZ\Publish\Core\MVC\ConfigResolverInterface::{hasParameter,getParameter}` method calls.
+
+Suggested results take into account namespace argument, if its value can be resolved without running interpreter
+(for example, string literal or const reference).
+
+### Intentions and inspections
+
+The plugin also brings several new intentions and inspections (with related quick fixes where possible).
+
+For example, when plugin detects deprecated configuration key usage, it marks the key as deprecated and suggests a replacement:
+
+![Intention](img/phpstorm_plugin_intention.png)
```

Comment:
It would be good idea to little modify the screenshot because when clicked, it is still too small to notice the feature.

Reply:
@adamwojs can you come to the rescue?